python/lsst/ts/ess/epm/modbus_simulator.py

```diff
@@ -47,23 +48,29 @@ class ModbusSimulator:
 
     def __init__(
         self,
+        log: logging.Logger,
         host: str = "0.0.0.0",
         port: int = 502,
         modbus_server: str = "server",
         modbus_device: str = "device",
     ) -> None:
+        self.log = log
```

Comment:
Please create a child logger like this:
```self.log = log.getChild(type(self).__name__)```
like you did in the AGC150 connector. That way it is clear which class logged in EFD.